src/components/TeamsPricingCard.astro

```diff
@@ -0,0 +1,76 @@
+---
+import Tooltip from "./Tooltip.astro";
+
+export interface Props {
+  title: string;
+  description: string;
+  price: string;
+  price_details: string;
+  payment_details: string;
+  features: string[];
+  feature_tooltip: string[];
+}
+
+const {
+  title,
+  description,
+  price,
+  price_details,
+  payment_details,
+  features,
+  feature_tooltip,
+} = Astro.props;
+---
+
+<div
+  class="bg-gradient-to-br from-blue-600 to-brand-pink w-full sm:w-[360px] sm:h-[572px] rounded-[32px] p-[4px]"
+>
+  <div class="h-full w-full rounded-[28px] bg-gray-800 px-8 py-12">
+    <div class="flex flex-col justify-between h-full">
+      <div class="space-y-6 sm:space-y-8">
+        <div>
+          <h3 class="text-xl sm:text-2xl font-semibold text-yellow-600">
+            {title}
+          </h3>
+          <p class="text-lg font-[100] font-['Inter'] text-gray-200 sm:h-16">
+            {description}
+          </p>
+        </div>
+        <div class="sm:h-20">
+          <h3 class="text-xl sm:text-2xl font-semibold text-white mt-4">
+            {price}
+            <span class="text-lg text-gray-400">{price_details}</span>
+          </h3>
+          <span
+            class="text-gray-200 text-sm italic font-['Inter'] leading-normal whitespace-normal font-light"
+            >{payment_details}</span
+          >
+        </div>
+        <ul>
+          {
+            features.map((feature, index) => (
+              <Tooltip
+                text={feature_tooltip[index]}
+                direction="top"
+                distant={false}
+              >
+                <li class="text-gray-300 text-lg font-light font-['Inter'] leading-loose">
+                  -> {feature}
+                </li>
+              </Tooltip>
+            ))
+          }
+        </ul>
+      </div>
+      <div
+        class="w-full sm:w-[280px] h-15 px-[47px] py-3.5 bg-gradient-to-br from-blue-600 to-brand-pink rounded-xl justify-center items-center gap-2.5 inline-flex mt-6 sm:mt-0"
+      >
+        <span
+          class="text-slate-50 text-lg font-semibold font-['Inter'] leading-loose"
+        >
+          Try it for FREE
```

Comment:
This should be a link, @hugobarauna where should it point?

Reply:
Done in https://github.com/livebook-dev/livebook.dev/pull/109/commits/5ca88a98f4856d9682a143cbbf1acb42091822e1